homework/classes/zadania_4.3_4.5.py

```diff
@@ -0,0 +1,176 @@
+### Zadanie 4.3 | Pociąg
+import pytest
+
+
+class Train:
+    MAX_SPEED_UPDATE = 75
+
+    def __init__(self, speed: int, fuel_amount: float):
+        self.speed = speed
+        self.fuel_amount = fuel_amount
+
+    def desc(self) -> str:
+        return f'Moja predkość to {self.speed}. Mam jeszcze {self.fuel_amount} litrów paliwa.'
+
+    def __str__(self) -> str:
+        return self.desc()
+
+    def acceleration(self, increase_speed: int) -> (int, float):
+        """
+        Metoda do zwiększania prędkości pociągu
+
+        :param increase_speed:
+        :return:
+        """
+        old_speed = self.speed
+        fuel_cost = (self.speed + increase_speed - old_speed) * (old_speed / 100)
+        if (self.speed < self.speed + increase_speed < self.speed * (Train.MAX_SPEED_UPDATE / 100) + self.speed) and \
+                fuel_cost <= self.fuel_amount:
+            self.speed += increase_speed
+            self.fuel_amount -= fuel_cost
+        return self.speed, self.fuel_amount
+
+
+train1 = Train(10, 1000)
+train1.acceleration(5)
+print(train1)
+train1.acceleration(20)
+print(train1)
+train1.acceleration(8)
+print(train1)
+train1.acceleration(10)
+print(train1)
+
+
+class TestTrain:
+    def test_acceleration(self):
+        train1 = Train(10, 1000)
+        assert train1
+        train1.acceleration(5)
+        assert train1.desc() == 'Moja predkość to 15. Mam jeszcze 999.5 litrów paliwa.'
+        train1.acceleration(20)
+        assert train1.desc() == 'Moja predkość to 15. Mam jeszcze 999.5 litrów paliwa.'
+        train1.acceleration(8)
+        assert train1.desc() == 'Moja predkość to 23. Mam jeszcze 998.3 litrów paliwa.'
+        train1.acceleration(10)
+        assert train1.desc() == 'Moja predkość to 33. Mam jeszcze 996.0 litrów paliwa.'
+
+
+
+### Zadanie 4.4 | Zbiornik
+print('*' * 100)
+
+class Zbiornik:
+    def __init__(self, ilosc_wody: float, tempertaura: float):
+        self.ilosc_wody = ilosc_wody
+        self.tempertaura = tempertaura
+    def __str__(self):
```

Comment:
Nie ma linijki odstępu między metodami. Zrób sobie reformat pliku https://www.jetbrains.com/help/pycharm/reformat-and-rearrange-code.html#reformat_file